spec/features/admin_deletes_user_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'rails_helper'
+
+feature 'admin deletes user profile', %Q{
+  As an admin, I want to view a list of all users, so that I can delete their
+  accounts
+  Acceptance Criteria:
+  * [X] - I can delete a user’s account.
+  } do
+
+  let(:admin) do
+    FactoryGirl.create(:user, admin: true)
+  end
+
+  before(:each) do
+    @user = FactoryGirl.create(:user)
+  end
+
+  scenario 'admin deletes user profile' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.